### PR TITLE
feat: dem quantized mesh

### PR DESCRIPTION
**Pull request by EllaMartirosyan**
## Checklist
- [x] Tests
- [x] Stories
- [x] Lint
- [x] Prettier

## What I did

- [ ] BREAKING CHANGE
- [x] feature request
- [ ] bug fix
- [ ] documentation

### CODE REVIEW

File: packages/react-components/src/lib/cesium-map/layers/3d.tileset.stories.tsx [hunk 1]
```diff
@@ -87,7 +87,6 @@ Cesium3DTilesetOnHeightLayer.argTypes = {
 export const CesiumSolar3DTilesetLayer: Story = (args: unknown) => (
   <div style={mapDivStyle}>
     <CesiumMap {...args}>
-      <Cesium3DTileset url="/mock/Rehovot_solar_tileset/L16_31232/L16_31232.json" />
       <Cesium3DTileset
         isZoomTo={true}
         url="/mock/Rehovot_solar_tileset/L16_31023/L16_31023.json"
```
Comment (alebinson) [inline packages/react-components/src/lib/cesium-map/layers/3d.tileset.stories.tsx]: can you remove 'rehovot' word

File: packages/react-components/src/lib/cesium-map/map.tsx [hunk 2]
```diff
@@ -117,6 +119,10 @@ export const CesiumMap: React.FC<CesiumMapProps> = (props) => {
     CesiumSceneModeEnum[] | undefined
   >();
   const [baseMaps, setBaseMaps] = useState<IBaseMaps | undefined>();
+  // eslint-disable-next-line
+  const [terrainProvider, setTerrainProvider] = useState<
```
Comment (alebinson) [inline packages/react-components/src/lib/cesium-map/map.tsx]: do you use **terrainProvider** somewhere?

File: packages/react-components/src/lib/cesium-map/terrain-providers/terrain-provider.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,229 @@
+import React, { useState } from 'react';
```
Comment (alebinson) [inline packages/react-components/src/lib/cesium-map/terrain-providers/terrain-provider.stories.tsx]: fix lin errors

**Review by alebinson — CHANGES_REQUESTED** [review]
Fix lint and prettier errors

File: packages/react-components/src/lib/cesium-map/layers/3d.tileset.stories.tsx (hunk 1 above)
Comment (EllaMartirosyan) [reply packages/react-components/src/lib/cesium-map/layers/3d.tileset.stories.tsx]: Removed

File: packages/react-components/src/lib/cesium-map/terrain-providers/terrain-provider.stories.tsx (hunk 3 above)
Comment (EllaMartirosyan) [reply packages/react-components/src/lib/cesium-map/terrain-providers/terrain-provider.stories.tsx]: Fixed

File: packages/react-components/src/lib/cesium-map/map.tsx (hunk 2 above)
Comment (EllaMartirosyan) [reply packages/react-components/src/lib/cesium-map/map.tsx]: Removed

**Review by alebinson — COMMENTED** [review]
gr8!

**Pull request merged**